GUI/XLSearchTaskWindow.xaml

```diff
@@ -81,7 +81,7 @@
                             <ComboBox x:Name="cbbXLprecusorMsTl" />
                         </StackPanel>
                         <StackPanel Orientation="Horizontal" Grid.Row="1" Grid.Column="1" Margin="5">
-                            <CheckBox x:Name="ckbXLTopNum" VerticalAlignment="Center" Content="Top N peptides per precursor" />
+                            <Label Content="Keep top N candidates" />
```

Comment:
Does this really need to be user settable?

Reply:
I prefer it to be user settable now. This may vary with different datasets. I may able to get a good number to suit all the datasets.

Reply:
I prefer it to be user settable now. This may vary with different datasets. I may able to get a good number to suit all the datasets.